etcdserver/server.go

```diff
@@ -1292,8 +1294,6 @@ func (s *EtcdServer) apply(es []raftpb.Entry, confState *raftpb.ConfState) (appl
 func (s *EtcdServer) applyEntryNormal(e *raftpb.Entry) {
 	shouldApplyV3 := false
 	if e.Index > s.consistIndex.ConsistentIndex() {
-		// set the consistent index of current executing entry
-		s.consistIndex.setConsistentIndex(e.Index)
```

Comment:
The ConsistentIndex tracks the raft index, the AppliedIndex tracks the last index that has been completely applied to the store. The applied index is necessarily <= the raft index since raft is where the data for the apply is coming from.